source/material_model/visco_plastic.cc

```diff
@@ -463,6 +496,13 @@ namespace aspect
                              "The extremely large default cohesion value (1e20 Pa) prevents the viscous stress from "
                              "exceeding the yield stress. Units: $Pa$.");
 
+          // Stress limiter parameters
+          prm.declare_entry ("Stress limiter exponents", "1.0",
+                             Patterns::List(Patterns::Double(0)),
+                             "List of stress limiter exponents, $n_lim$, for background material and compositional fields, "
```

Comment:
The line is too long to see it on github, which likely means that it's also generally too long, I suppose :-)